service/src/main/resources/sql/mysql/celeborn-0.5.0-mysql.sql

```diff
@@ -54,3 +54,15 @@ CREATE TABLE IF NOT EXISTS celeborn_cluster_tenant_config
     PRIMARY KEY (id),
     UNIQUE KEY `index_unique_tenant_config_key` (`cluster_id`, `tenant_id`, `name`, `config_key`)
 );
+
+CREATE TABLE IF NOT EXISTS celeborn_cluster_tags
```

Comment:
Do we need to create new sql shell for 0.6.0?

Reply:
Hi @SteNicholas, did not understand what do mean by "create new sql shell", is there any sql utility that we need to change? But as mentioned in the description, this will introduce a user facing change that they will have to create this new table for storing tags.

Reply:
@s0nskar, this sql file is created for 0.5.0 not for 0.6.0. Do we need to create new sql file for 0.6.0?

Reply:
Agreed, it will better to keep separate files by version. Tests are also using 0.5.0 file, IMO for tests we can rename the file to 0.6.0 . WDYT?

Reply:
You'd better to create new sql file for 0.6.0 and keep the current sql file for 0.5.0.

Reply:
Thanks @SteNicholas for the suggestion, created two separate sql files for 0.5.0 and 0.6.0 and a upgrade script to upgrade from 0.5.0 to 0.6.0. Also added a small README. PTAL.